LICENSE

```diff
@@ -50,4 +50,4 @@ You may add Your own copyright statement to Your modifications and may provide a
 
 9. Accepting Warranty or Additional Liability. While redistributing the Work or Derivative Works thereof, You may choose to offer, and charge a fee for, acceptance of support, warranty, indemnity, or other liability obligations and/or rights consistent with this License. However, in accepting such obligations, You may act only on Your own behalf and on Your sole responsibility, not on behalf of any other Contributor, and only if You agree to indemnify, defend, and hold each Contributor harmless for any liability incurred by, or claims asserted against, such Contributor by reason of your accepting any such warranty or additional liability.
 
-END OF TERMS AND CONDITIONS
```

Comment:
I cannot see a reason why this file is touched. I'm assuming there is a whitespace difference, but I would prefer to just not have the license touched.

Reply:
I ensured the files are exactly the same. Not sure why a diff even shows up. Let me know if this is important. I think I can do some GIT gymnastics to just remove this file from the commit.

Reply:
I'd be happier if you could do the git gymnastics to remove the file from the commit.

Reply:
The easiest thing is for Ed to do another PR excluding the LICENSE file. Doing that now and closing this PR.